### PR TITLE
Pytest: new recipe

### PR DESCRIPTION
**Pull request by Oyami-Srk**
A Python testing framework and its dependencies.

### CODE REVIEW

File: dev-python/funcsigs/funcsigs-1.0.2.recipe [hunk 1]
```diff
@@ -0,0 +1,58 @@
+SUMMARY="Python signature tools"
+DESCRIPTION="Python function signatures from PEP362 for Python 2.6, 2.7 and 3.2+."
+HOMEPAGE="http://funcsigs.readthedocs.org"
+SOURCE_URI="https://pypi.python.org/packages/94/4a/db842e7a0545de1cdb0439bb80e6e42dfe82aaeaadd4072f2263a4fbed23/funcsigs-1.0.2.tar.gz"
+REVISION="1"
+LICENSE="Apache v2"
+COPYRIGHT="Testing Cabal" CHECKSUM_SHA256="a7bb0f2cf3a3fd1ab2732cb49eba4252c2af4240442415b4abce3b87022a8f50"
```
Comment (aarroyoc) [inline dev-python/funcsigs/funcsigs-1.0.2.recipe]: CHECKSUM_SHA256 should be in a new line

**Comment by aarroyoc** [comment]
Ok, the Travis CI test is failing because you have introduced a trailing whitespace in the funcsigs recipe. Remove it first

**Comment by aarroyoc** [comment]
+1 for me

**Comment by Oyami-Srk** [comment]
Thank you for your helping!

File: dev-python/py/py-1.5.2.recipe [hunk 2]
```diff
@@ -0,0 +1,61 @@
+SUMMARY="A Python development support library"
+DESCRIPTION="The py lib is a Python development support library featuring the following tools and modules:."
```
Comment (pulkomandy) [inline dev-python/py/py-1.5.2.recipe]: Did the list of tools and modules get lost?

File: dev-python/py/py-1.5.2.recipe [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+HOMEPAGE="https://pytest.org"
+SOURCE_URI="https://pypi.python.org/packages/90/e3/e075127d39d35f09a500ebb4a90afd10f9ef0a1d28a6d09abeec0e444fdd/py-1.5.2.tar.gz#md5=279ca69c632069e1b71e11b14641ca28"
+REVISION="1"
+LICENSE="MIT"
+COPYRIGHT="Holger Krekel and others. 2004-2017"
```
Comment (pulkomandy) [inline dev-python/py/py-1.5.2.recipe]: "2004-2017 Holger Krekel and others."

File: dev-python/funcsigs/funcsigs-1.0.2.recipe [hunk 4]
```diff
@@ -0,0 +1,60 @@
+SUMMARY="Python signature tools"
+DESCRIPTION="Python function signatures from PEP362 for Python 2.6, 2.7 and 3.2+."
+HOMEPAGE="http://funcsigs.readthedocs.org"
+SOURCE_URI="https://pypi.python.org/packages/94/4a/db842e7a0545de1cdb0439bb80e6e42dfe82aaeaadd4072f2263a4fbed23/funcsigs-1.0.2.tar.gz"
+REVISION="1"
+LICENSE="Apache v2"
+COPYRIGHT="Testing Cabal"
```
Comment (scottmc) [inline dev-python/funcsigs/funcsigs-1.0.2.recipe]: COPYRIGHT should have year or year range listed.  Perhaps this should be "2013 Aaron Iles" ?
https://github.com/testing-cabal/funcsigs/blob/master/LICENSE

**Pull request merged**